tsdb/querier.go

```diff
@@ -141,6 +142,7 @@ type blockChunkQuerier struct {
 }
 
 // NewBlockChunkQuerier returns a chunk querier against the block reader and requested min and max time range.
+// It's caller responsibility to close ChunkQuerier to release resources (e.g close chunk reader)
```

Comment:
I'd stop the sentence after resources, how this is implemented at lower levels doesn't belong in the function docs.